src/app/leverage/utils/get-leverage-base-token.ts

```diff
@@ -8,7 +8,9 @@ const uSui = getTokenByChainAndSymbol(base.id, 'uSUI')
 
 export function getLeverageBaseToken(symbol: string) {
```

Comment:
I'm thinking that the `getUnderlyingToken` function from the tokenList might be able to help here but just a recommendation, im not 100% sure